d3b_utils/requests_retry.py

```diff
@@ -53,14 +59,21 @@ def __init__(
             backoff_factor=backoff_factor,
             status_forcelist=status_forcelist,
             method_whitelist=method_whitelist,
+            raise_on_status=False,
             **kwargs,
         )
         adapter = HTTPAdapter(max_retries=retry)
         self.mount("http://", adapter)
         self.mount("https://", adapter)
 
     def send(self, req, **kwargs):
-        self.logger.debug(f"Sending request: {vars(req)}")        
+        self.logger.debug(f"Sending request: {vars(req)}")
         resp = super().send(req, **kwargs)
         self.logger.debug(f"Got response:  {vars(resp)}")
+        try:
+            if resp.status_code in self.status_forcelist:
+                resp.raise_for_status()
+        except HTTPError:
+            self.logger.debug(f"MAX RETRIES EXCEEDED. Error body:  {resp.text}")
```

Comment:
Are you sure this (max retries exceeded) is the only HTTP error which doesn't include the resp body? If not, I would just remove the `MAX RETRIES EXCEEDED` part.